tests/integration/message/message.spec.ts

```diff
@@ -48,14 +48,14 @@ describe('Message Integration Tests - Happy Path', () => {
     });
   });
 
-  // TODO: Extend those three tests later on
   describe('#getMessages', () => {
     it('should return all messages if no query params', async () => {
       await requestSender.createMessage({ requestBody: fullMessageInstance });
       const response = await requestSender.getMessages();
 
       expect(response).toSatisfyApiSpec();
       expect(response.status).toBe(httpStatusCodes.OK);
+      expect(response.body).not.toEqual([]);
     });
 
     it('should return filtered messages', async () => {
```

Comment:
'should return filtered messages' - where do you check that the filtered results match the one you requested?
steps in this test
1. get - no results
2. add new
3. get - with result